programs/steward/src/state/large_bitmask.rs

```diff
@@ -0,0 +1,96 @@
+use anchor_lang::{prelude::Result, zero_copy};
```

Comment:
Doc comment how this file is a copy of `bitmask.rs`, copied because of restrictions around variable size data structures in zero copy accounts and overhead of function calls if we had shared logic

Reply:
Done

```
/*
    This file is largely copied over from bitmask.rs
    This is because making a generic bitmask struct either didn't play well with
    zero-copy, or it added too much overhead to a struct meant for performance.
*/
```